android/package.json

```diff
@@ -15,13 +15,13 @@
 	],
 	"minSDKVersion": "16",
 	"vendorDependencies": {
-		"android sdk": ">=23 <=25.x",
-		"android build tools": ">=23 <=25.x",
-		"android platform tools": ">=17 <=25.x",
+		"android sdk": ">=25 <=26.x",
+		"android build tools": ">=25 <=26.x",
+		"android platform tools": ">=25 <=26.x",
 		"android tools": "<=26.x",
-		"android ndk": ">=r8e <=r9",
+		"android ndk": ">=r11c <=r15c",
```

Comment:
Regarding Android NDK version, we have to watch out for C++ ABI compatibility issues with our precompiled V8 library. This is to avoid compiler errors and might impact hyper loop. We can figure this out later. (This is more of a warning.)